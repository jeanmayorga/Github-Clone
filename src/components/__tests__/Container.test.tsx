import { render } from '@testing-library/react'
import { Container } from '..'

describe('Render container', () => {
  it('should render container', () => {
    const { container } = render(
      <Container>
        <div>Im a children</div>
      </Container>
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
