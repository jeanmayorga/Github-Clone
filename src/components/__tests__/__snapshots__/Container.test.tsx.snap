// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render container should render container 1`] = `
<div
  class="w-[1280px] m-auto px-[32px]"
>
  <div>
    Im a children
  </div>
</div>
`;
